src/win/util.c

```diff
@@ -484,7 +484,7 @@ uint64_t uv__hrtime(double scale) {
    * performance counter interval, integer math could cause this computation
    * to overflow. Therefore we resort to floating point math.
    */
-  return (uint64_t) ((double) counter.QuadPart * hrtime_interval_ * scale);
+  return (uint64_t) (counter.QuadPart * (uint64_t) scale / hrtime_interval_);
```

Comment:
@piscisaureus , I think you are thinking about this comment: https://github.com/joyent/libuv/pull/1165#issuecomment-52793171

Basically, QPC won't wrap in our lifetimes. But using integer math is the wrong approach IMO, and can introduce overflow and rounding errors.